ci-operator/templates/openshift/installer/cluster-launch-installer-ovirt-e2e.yaml

```diff
@@ -138,7 +138,7 @@ objects:
             conformance_resource_name="$(echo "${conformance_resource}"|jq .name --raw-output)"
             echo "[INFO] Conformance Lease acquired! at $(date --utc)"
             echo "[INFO] Conformance Leased resource: ${conformance_resource}"
-            boskosctl --server-url http://boskos.ci --owner-name "${CLUSTER_NAME}" heartbeat --resource "${resource}" &
+            boskosctl --server-url http://boskos.ci --owner-name "${CLUSTER_NAME}" heartbeat --resource "${conformance_resource}" &
```

Comment:
should this be `conformance_resource_name`?

Reply:
yes

Reply:
not based on https://github.com/openshift/release/pull/8311/files#diff-fdd0a32995450498fb5ce86729e19eecR133